src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1219,37 +1220,53 @@ private boolean isSuspended()
             private int srcSeqNumStart = -1;
 
             /**
+             * Update this {@link SeenFrame} with information from its
+             * corresponding {@link FrameDesc}
+             * @param frame the {@link FrameDesc} from which to update
+             */
+            void update(FrameDesc frame)
+            {
+                if (this == mostRecentSentFrame /* the max frame can expand */
+                    || availableQualityIndices == null
+                    || availableQualityIndices.length < 2)
+                {
+                    if (frame.lastSequenceNumberKnown())
+                    {
+                        srcSeqNumLimit = frame.getEnd();
+                    }
+                    else
+                    {
+                        srcSeqNumLimit = frame.getMaxSeen();
+                    }
+                }
+                if (frame.firstSequenceNumberKnown())
```

Comment:
Again a tiny change of semantics: with the old code, if srcSeqNumLimit is -1, then srcSeqNumStart is not updated (line 1246). Not saying this is a problem, just asking if it's has been considered.